spring-cloud-zookeeper-discovery/src/main/java/org/springframework/cloud/zookeeper/discovery/ZookeeperServerList.java

```diff
@@ -74,7 +76,7 @@ private String getServiceIdFromDepsOrClientName(IClientConfig clientConfig, Zook
 	}
 
 	@SuppressWarnings("unchecked")
-	private List<ZookeeperServer> getServers() {
+	protected List<ZookeeperServer> getServers() {
```

Comment:
If a user wants to change the conditions of when a `ServiceInstance` gets added I thought it would be better to make this protected, than having to copy everything into a new class.